### PR TITLE
feat: testing

### PR DESCRIPTION
**Pull request by johnny243**
add tests for the following:

- Evernote to Standard Notes
- Google Keep to Standard Notes
- Plain text to Standard Notes
- Simplenote to Standard Notes

### CODE REVIEW

File: lib/sntools.js [hunk 1]
```diff
@@ -365,76 +365,72 @@ class SNTools {
...
   }
 
-  convertSimplenoteFiles(files, completion) {
-    let index = 0;
-    let processedData = [];
-
-    let tags = new Map();
+  convertSimplenoteFiles(rawNotes) {
+    // Final notes array
+    let notes = [];
```
Comment (mobitar) [inline lib/sntools.js:370]: Could be const? Also we can rename `notes` to `finalNotes` and remove the above comment.

File: lib/sntools.js [hunk 2]
```diff
@@ -223,18 +227,13 @@ class SNTools {
...
-        title = element.getElementsByTagName('title')[0].innerText;
+        title = element.getElementsByClassName('title')[0].innerHTML;
       } catch (e) {
         // Invalid note, continue
         console.log(note.name, 'is an invalid note (no title)');
         continue;
       }
 
-      // Check if title is date (default if no title is set). If so, use empty string
-      if (title !== '' && !isNaN(new Date(title))) {
```
Comment (mobitar) [inline lib/sntools.js:234]: Is this no longer needed? What does EN do with as default title?

**Comment by mobitar** [comment]
Can you check if [this issue](https://github.com/standardnotes/forum/issues/587#issuecomment-821754550) is solved with these changes?

File: lib/sntools.js (hunk 2 above)
Comment (johnny243) [reply lib/sntools.js:234]: I added fallback values for both EN and GK titles. It's better than having a note with a blank title or dates as titles IMO.

**Comment by johnny243** [comment]
[Formatting issue](https://github.com/standardnotes/forum/issues/587#issuecomment-821754550) fixed here: https://github.com/standardnotes/sntools/pull/27/commits/07e1b7c4c4ebaf7b4e8483ea453e58c3679f4b25

**Review by mobitar — APPROVED** [review]

**Pull request merged**